### ISSUE TITLE
Add Firefox support

### ISSUE THREAD

**Issue by russellbanks** [NirmalScaria/le-git-graph#11]
As title, it would be nice if Firefox was supported :)

**Comment by NirmalScaria** [comment]
Hey,
I am actively working on it, but the extension is built with manifest v3 and firefox doesn't yet support manifest v3 (as far as i know.)
It might take some time to make enough modifications to have a manifest v2 based, firefox compatible version of the extension.
Contributions on this are highly appreciated.

**Comment by Naatan** [comment]
Manifest v3 is available with a config tweak apparently:

https://extensionworkshop.com/documentation/develop/manifest-v3-migration-guide/

I'd say the general audience that would be interested in your extension would be comfortable toggling that setting. Developing it for manifest v2 and then migrating to v3 might not be worth it considering how close v3 appears to be.

**Comment by choma** [comment]
Manifest v3 signing starts today:

https://www.theregister.com/2022/11/17/mozilla_next_week_will_begin/

**Comment by NirmalScaria** [comment]
Hey,
I have published the extension for firefox. I tried manifest v3 in firefox but that didn't work quite well, and there were not enough resources. Making few changes to adapt for manifest v2 was easier and so... firefox version is based on manifest v2, at least temporarily, until MV3 becomes stable for fireefox.

Extension got published on firefox today. Hope it becomes useful. 😃

https://addons.mozilla.org/en-US/firefox/addon/le-git-graph-github-git-graph/
